### ISSUE TITLE
one bad recipe

### ISSUE THREAD

**Issue by moonfather1** [HydraOC/More-Tetra-Materials#14]
soul steel ingot has a blasting recipe using non-existant item. not a big deal, just a reminder.

**Comment by HydraOC** [comment]
Yeah, ditched the recipe halfway through development of it and went for using lightning, keep forgetting to remove the json
